### DISCUSSION TITLE
Where can I find the variables for color and sizing etc. used in the app.css?

### DISCUSSION THREAD

**Discussion by GlacierFox** [microsoft/fluentui-blazor#1747] [Q&A] [answered]
Hi, I'm looking for where to find the variables for colors and other miscellaneous values that are used in the app.css so I can use them in my own classes to keep things consistent.
I an see things like 
`color: var(--neutral-foreground-rest);` , 

`var(--type-ramp-base-line-height);` and

` var(--accent-fill-rest); `
but I'm not sure where they're coming from. 

Thanks!

**Comment by vnbaaij** [comment, marked as answer]
These are set by the DesignTokens coming out of the underlying Fluent UI Web Components script. See https://www.fluentui-blazor.net/DesignTokens. Basically evry name you find ther translates to a CSS variable with (-- in front etc)
